packages/peregrine/src/hooks/__tests__/useWindowSize.spec.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { act } from 'react-test-renderer';
+
+import { useWindowSize } from '../useWindowSize';
+import createTestInstance from '../../util/createTestInstance';
```

Comment:
This test is based on @jimbo's work in #1078. Until that's merged these tests won't work.